Sources/CustomEvent/CustomEvent.swift

```diff
@@ -1,25 +1,20 @@
 //
 // CustomEvent.swift
 // Kommunicate
-//Created by Sathyan Elangovan on 09/12/21.
+// Created by Sathyan Elangovan on 09/12/21.
 
 import Foundation
 
 public enum CustomEvent: String {
     case messageSend = "ON_MESSAGE_SEND"
     case faqClick = "ON_FAQ_CLICK"
-    case notificationClick = "ON_NOTIFICATION_CLICK"
     case newConversation = "ON_START_NEW_CONVERSATION_CLICK"
-    case attachmentClick = "ON_ATTACHMENT_ICON_CLICK"
-    case voiceClick = "ON_VOICE_ICON_CLICK"
-    case locationClick = "ON_LOCATION_ICON_CLICK"
-    case rateConversationClick = "ON_RATE_CONVERSATION_CLICK"
-    case rateConversationEmotionsClick = "ON_RATE_CONVERSATION_EMOTIONS_CLICK"
     case submitRatingClick = "ON_SUBMIT_RATING_CLICK"
     case restartConversationClick = "ON_RESTART_CONVERSATION_CLICK"
     case richMessageClick = "ON_RICH_MESSAGE_CLICK"
-    case resolveConversation = "ON_CONVERSATION_RESOLVE"
     case conversationBackPress = "ON_CONVERSATION_BACK_CLICK"
     case conversationListBackPress = "ON_CONVERSATION_LIST_BACK_CLICK"
     case messageReceive = "ON_MESSAGE_RECEIVE"
+    // TODO: ADD THESE EVENTS(attachmentClick,voiceClick, locationClick, rateConversationEmotionsClick) WITH PROPER DATA
+    // TODO: FIX FLOW: resolveConversation
```

Comment:
We can file issues for this and remove from here if they are not needed.

Reply:
removed & filed issue